felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -3254,6 +3254,9 @@ func (m *bpfEndpointManager) OnHEPUpdate(hostIfaceToEpMap map[string]*proto.Host
 }
 
 func (m *bpfEndpointManager) addHEPToIndexes(ifaceName string, ep *proto.HostEndpoint) {
+	if ep == nil {
```

Comment:
Looks like `removeHEPFromIndexes` uses a different fix (it calls `GetTiers()` et al, which handle the `nil`).  Please can you bring them into sync?

I think I find your way a bit clearer (and less subject to an external change breaking us).